README.md

```diff
@@ -42,9 +23,33 @@ By default, no Pointer Events are sent from an element. This maximizes possibili
 
 1. As elements come and go, or have their `touch-action` attribute changed, they will send the proper set of Pointer Events.
 
-See also the [examples in the W3C Pointer Events specification](http://www.w3.org/TR/pointerevents/#examples).
+```html
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>PEP (Pointer Events Polyfill)</title>
+    <meta name="viewport" content="width=device-width">
+    <!-- include PEP -->
+    <script src="https://code.jquery.com/pep/0.4.0/pep.js"></script>
```

Comment:
This will need an accompanying change to the release script to update the version in the README, otherwise this will get out of date.